Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [profile.release]
 panic = "unwind"
+opt-level = 3
```

Comment:
What does this mean?

Reply:
https://doc.rust-lang.org/cargo/reference/profiles.html#opt-level

> The opt-level setting controls the [-C opt-level flag](https://doc.rust-lang.org/rustc/codegen-options/index.html#opt-level) which controls the level of optimization. Higher optimization levels may produce faster runtime code at the expense of longer compiler times. Higher levels may also change and rearrange the compiled code which may make it harder to use with a debugger.
> 
> The valid options are:
> 
> 0: no optimizations
> 1: basic optimizations
> 2: some optimizations
> 3: all optimizations
> "s": optimize for binary size
> "z": optimize for binary size, but also turn off loop vectorization.